### ISSUE TITLE
Save bar in Arc browser disappears too quickly

### ISSUE THREAD

**Issue by Amparose** [omnivore-app/omnivore#2856]
After saving an article, the Omnivore bar to add labels, etc, goes away far too quickly to interact with it. I think **it shouldn't dismiss itself automatically** at all and instead be closed by the user; there's already an "x" button on the right (not that I've ever gotten the mouse there quick enough to try it).

**Comment by jacksonh** [comment]
Right now its set to 5 seconds and then pauses if interacted with. I wonder if Arc is removing it faster than that? 

We could probably add something in options.

**Comment by Amparose** [comment]
Perhaps it's because I'm now aware of the timeout, but the last few saves didn't seem as crazy-fast. I have restarted the browser since though so not sure.

I would like to make this a feature request for the **option to not timeout and instead manually dismiss**. I can imagine saving an article while still browsing it and deciding what labels to apply etc.

**Comment by jacksonh** [comment]
Yeah I think adding as an option is pretty simple and makes a lot of sense.

**Comment by jacksonh** [comment]
https://github.com/omnivore-app/omnivore/pull/2873

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
